Editor/Resources/EditorWindow/Pages/AwsUserProfilesPage.uxml

```diff
@@ -39,15 +40,18 @@
             <ui:VisualElement class="separator separator--horizontal separator--centered">
                 <ui:Button name="UserProfilePageBootstrapStartButton" class="button button--primary" text="Bootstrap Profile"/>
                 <ui:Button name="UserProfilePageBootstrapAnotherProfileButton" class="button" text="Add another Profile"/>
-                <ui:Label name="UserProfilePageBootstrapHelpLink" class="link" text="What is bootstrapping?  \uf08e"/>
+                <ui:VisualElement name="UserProfilePageBootstrapHelpLink" class="external-link float-right">
+                    <ui:Label text="What is bootstrapping?"/>
+                    <ui:Image name="ExternalLinkIcon" class="icon--small"/>
+                </ui:VisualElement>
             </ui:VisualElement>
         </ui:VisualElement>
         <ui:VisualElement name="UserProfilePageCompletedMenu" class="separator separator--large">
             <gl:ProfileSelector/>
             <ui:VisualElement class="form-row separator separator--horizontal">
                 <ui:Button name="UserProfilePageBootstrapAnotherBucketButton" class="button" text="Bootstrap to new S3 Bucket"/>
                 <ui:Button name="UserProfilePageAccountAddNewProfileButton" class="button" text="Add another Profile"/>
-                <ui:VisualElement class="external-link float-right">
+                <ui:VisualElement name="UserProfilePageCompletedBootstrapHelpLink" class="external-link float-right">
                     <ui:Label text="What is bootstrapping?"/>
```

Comment:
Same for this piece of text. It needs to come from the TextProvider and shouldn't be hardcoded

Reply:
Solved